### PR TITLE
Feature/iia 799 mock modal consistent

### PR DESCRIPTION
**Pull request by dholubek**
image
image

combining the change set export and the fhir export into one screen

### CODE REVIEW

File: application/src/main/java/dev/ikm/komet/app/App.java [hunk 1]
```diff
@@ -321,12 +310,11 @@ public void start(Stage stage) {
             importDatasetMenuItem.setOnAction(actionEvent -> doImportDataSet(primaryStage));
 
             // Exporting data
-            Menu exportMenu = new Menu("Export Dataset");
-            MenuItem fhirMenuItem = new MenuItem("FHIR");
-            fhirMenuItem.setOnAction(actionEvent -> openDatasetPage());
-            exportMenu.getItems().addAll(createExportChangesetMenuItem(), fhirMenuItem);
+                        MenuItem exportDatasetMenuItem = new MenuItem("Export Dataset");
```
Comment (carldea) [inline application/src/main/java/dev/ikm/komet/app/App.java:313]: Why is this spaced oddly?

Comment (dholubek) [reply application/src/main/java/dev/ikm/komet/app/App.java:313]: I think a errand typo, will fix

File: application/src/main/java/dev/ikm/komet/app/App.java [hunk 2]
```diff
@@ -810,30 +798,27 @@ private void launchClassicKomet() throws IOException, BackingStoreException {
...
-            datasetPageController.setViewProperties(windowSettings.getView().makeOverridableViewProperties());
-            Scene sourceScene = new Scene(datasetBorderPane, 550, 700);
-            datasetStage.setScene(sourceScene);
-            datasetStage.setTitle("Export Dataset");
-            datasetStage.show();
-        } catch (IOException e) {
-            throw new RuntimeException(e);
-        }
+        Stage exportStage = new Stage();
+        //set up ExportViewModel
```
Comment (carldea) [inline application/src/main/java/dev/ikm/komet/app/App.java:812]: Can we make the Stage window modal?
```java
exportStage.initOwner(parentStage);
exportStage.initModality(Modality.APPLICATION_MODAL); 
...// rest of code

exportStage.showAndWait(); // move or replace this at line 821 
```

Comment (carldea) [reply application/src/main/java/dev/ikm/komet/app/App.java:812]: Actually, we can approve and create another ticket to resolve this scenario "Modality".

**Pull request merged**